spec/directions/move_title_text_into_span_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::MoveTitleTextIntoSpan do
+
+  let(:element_1) do
```

Comment:
Make this be `let(:title)` and then you can get rid of `title = element_1` below